source/extensions/transport_sockets/alts/config.cc

```diff
@@ -123,10 +123,10 @@ Network::TransportSocketFactoryPtr createTransportSocketFactoryHelper(
     tsi_handshaker* handshaker = nullptr;
     // Specifying target name as empty since TSI won't take care of validating peer identity
     // in this use case. The validation will be performed by TsiSocket with the validator.
-    // Leaving the max frame size as 0 (unspecified) sets the default max frame size of 128 KiB.
+    // Set the max frame size to 16KB.
     tsi_result status = alts_tsi_handshaker_create(
         options.get(), target_name, handshaker_service.c_str(), is_upstream,
-        nullptr /* interested_parties */, &handshaker, 0 /* default max frame size */);
+        nullptr /* interested_parties */, &handshaker, 16384 /* default max frame size */);
```

Comment:
Thanks for making these changes.

Would it be possible to add some basic tests to cover the behavior changes in this PR?

Reply:
+1 on tests. Thanks @yihuazhang for the quick PR!

Reply:
It took some time figuring out how the current envoy/alts tests work. I added some tests to verify if a correct frame size (16KB) will be used by a protector. There are some tests that I want to add, which depend on the change in gRPC code base (https://github.com/grpc/grpc/pull/25621). I added TODO for them, and will fix them once the PR is in and envoy updates the gRPC version.